addons/binding/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/handler/AVMFritzBaseThingHandler.java

```diff
@@ -63,9 +65,12 @@ public AVMFritzBaseThingHandler(Thing thing) {
         super(thing);
     }
 
-    /**
-     * Handle the commands for switchable outlets or heating thermostats.
-     */
+    @Override
+    public void initialize() {
+        logger.debug("Initializing handler for {}", getClass().getName());
+        updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.NONE, "Waiting for refresh");
```

Comment:
Shouldn't this set the status to UNKNOWN here? I'd also remove the "Waiting for refresh"